.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -37,10 +37,11 @@ If applicable, add screenshots to help explain your problem.
 
 ### Checklist
 
-Make sure these boxes are checked before submitting your issue - thank you!
+Make sure to follow these steps before submitting your issue - thank you!
 
 - [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
-- [ ] I have reproduced the issue with at least the latest released version of superset.
+- [ ] I have reproduced the issue with at least the latest released version of superset. This issue has been reproduced on the following versions:
+  - `insert version here`
```

Comment:
oh yeah I don't know how I didn't notice that. I'll go ahead and remove that bit, and also remove the npm version bit since npm version doesn't really impact anything at runtime.